crossdock/src/driver.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+/node_modules/.bin/babel-node /crossdock/src/http_server.js &
+/node_modules/.bin/babel-node /crossdock/src/tchannel_server.js &
```

Comment:
we could, but why not start them each as a separate process?  This implementation has better performance technically.